openassessment/templates/openassessmentblock/staff_debug/student_info.html

```diff
@@ -25,12 +25,11 @@ <h3 class="title">{% trans "Student Response" %}</h3>
                     {% endif %}
                 </div>
 
-                {% if submission.image_url %}
```

Comment:
Apologies if I missed this elsewhere, but does this change mean that the staff debug area no longer shows the image preview, or was this moved to another template by chance?